app/main.py

```diff
@@ -1,25 +1,98 @@
 from abc import ABC
+from typing import Any
 
 
 class IntegerRange:
```

Comment:
You need to add __set__ to this class. This method should raise ValueError in case if there is wrong value to set.